GitUI/CommandsDialogs/RevisionFileTree.cs

```diff
@@ -252,12 +252,15 @@ private void LoadInTree(IEnumerable<IGitItem> items, TreeNodeCollection node)
                     {
                         continue;
                     }
-                    var fileIcon = _iconProvider.Get(Path.Combine(Module.WorkingDir, gitItem.FileName));
+                    var fileIcon = _iconProvider.Get(Module.WorkingDir, gitItem.FileName);
```

Comment:
Here the fileIcon is computed not knowing if it is needed. We know that for sure in line 262.